simple_app/plots.py

```diff
@@ -71,7 +72,10 @@ def normalise(fluxarr: np.ndarray) -> np.ndarray:
         spectrum: Spectrum1D = spec['spectrum']  # spectrum as an object
         if isinstance(spectrum, str):
             continue
-        wave: np.ndarray = spectrum.spectral_axis.to(u.micron).value  # unpack wavelengths
+        try:
+            wave: np.ndarray = spectrum.spectral_axis.to(u.micron).value  # unpack wavelengths
+        except astropy.units.UnitConversionError:
+            continue
```

Comment:
Provide some more info to user about this failure. Maybe just keep track of how many were not plotted.